src/cloudevent/mocks/helpers.ts

```diff
@@ -27,6 +28,15 @@ export function getBaseCloudEvent<EventType extends CloudEvent<unknown>>(
   } as EventType;
 }
 
+export function extractStringFromStringOrStringParam(
+  stringOrExpression: string | Expression<string>
+) {
+  if (typeof stringOrExpression === 'string') {
+    return stringOrExpression;
+  }
+  return stringOrExpression?.value();
```

Comment:
Probably double check with @Berlioz - but I thought it is very possible that this to return empty string.

I'm wondering if we should think more about supporting Expressions firebase-functions. Expressions are a way to specify "value of this is going to be determined at deploy time" - so it is probably going to be undefined at test time unless there is some special override mechanism.

Reply:
The problem I was working against was `eventTrigger.eventFilters` and `eventTrigger.eventFilterPathPatterns` were converted to type `Record<string, string | Expression<string>>` :|

Reply:
The tests show a scenario where its populated at runtime too